BitwardenShared/Core/Tools/Models/Domain/PasswordGenerationOptions.swift

```diff
@@ -48,6 +48,9 @@ struct PasswordGenerationOptions: Codable, Equatable {
 
     /// The separator to put between words in the passphrase.
     var wordSeparator: String?
+
+    /// Whether the password type should be enforced or not
+    var overridePasswordType: Bool?
```

Comment:
🤔 Why is this boolean optional? What does a nil value represent as different from a false value?

Reply:
should never be nil actually, removed the optional

Reply:
Actually, this is making the [test_decode_empty](https://github.com/bitwarden/ios/blob/235f340f670d02f279a2409ec9bf596f63f888a3/BitwardenShared/Core/Tools/Models/Domain/PasswordGenerationOptionsTests.swift#L54) on `PasswordGenerationOptionsTests` to fail.

I'm afraid it might throw an exception when calling the [decodePasswordOptions](https://github.com/bitwarden/ios/blob/235f340f670d02f279a2409ec9bf596f63f888a3/BitwardenShared/Core/Autofill/Utilities/ActionExtensionHelper.swift#L201) and there is no `overridePasswordType`.

Reply:
This model is persisted to UserDefaults to maintain the user's options between app runs. The optional bool allows decoding to succeed even if it doesn't exist in the previously persisted model.